site/components/form/DateSelector.tsx

```diff
@@ -77,7 +77,15 @@ const DateSelector = <T extends object>({
     stateUpdater,
 }: DateSelectorProps<T>): ReactElement => {
     const [dateValue, setDateValue] = useState<Date | null>(!!disabled || !value ? null : new Date(value));
-    const [errors, setErrors] = useState<ErrorInfo[]>(initialErrors);
+    const errorsToUse = !disabled ? initialErrors : initialErrors.filter((error) => !(error.id === inputId));
```

Comment:
I don't think we should need this filter, the hook will run on the initial render so if disabled is true the errors will be cleared down by that

Reply:
Removed